README.md

```diff
@@ -305,6 +305,24 @@ You can deploy individual services to any cloud provider or self host them.
 
 The demo service (links above) is deployed to Fly.io (Go, SQLite) and Vercel (Next.js) and are under the free tiers.
 
+### Backend Development setup
+
+Run the following command for setting up backend for development purpose
+
+```
+make run
```

Comment:
should we add cd api here?

Reply:
sure,would add once the below conversations are resolved